src/test/java/uk/gov/hmcts/reform/unspec/service/tasks/handler/CaseEventTaskHandlerTest.java

```diff
@@ -6,44 +6,61 @@
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.InjectMocks;
 import org.mockito.Mock;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDataContent;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.ccd.client.model.StartEventResponse;
+import uk.gov.hmcts.reform.unspec.enums.BusinessProcessStatus;
+import uk.gov.hmcts.reform.unspec.helpers.CaseDetailsConverter;
+import uk.gov.hmcts.reform.unspec.model.BusinessProcess;
+import uk.gov.hmcts.reform.unspec.model.CaseData;
+import uk.gov.hmcts.reform.unspec.sampledata.CaseDataBuilder;
+import uk.gov.hmcts.reform.unspec.sampledata.CaseDetailsBuilder;
 import uk.gov.hmcts.reform.unspec.service.CoreCaseDataService;
 
 import java.util.Map;
 
-import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.eq;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 import static uk.gov.hmcts.reform.unspec.callback.CaseEvent.NOTIFY_RESPONDENT_SOLICITOR1_FOR_CLAIM_ISSUE;
 
+@SpringBootTest(classes = {
+    CaseEventTaskHandler.class,
+    JacksonAutoConfiguration.class,
+    CaseDetailsConverter.class
+})
 @ExtendWith(SpringExtension.class)
 class CaseEventTaskHandlerTest {
 
-    private static final String errorMessage = "Event failed processing";
     private static final Long CASE_ID = 1L;
 
     @Mock
     private ExternalTask mockExternalTask;
     @Mock
     private ExternalTaskService externalTaskService;
-    @Mock
+    @MockBean
     private CoreCaseDataService coreCaseDataService;
-    @InjectMocks
+    @Autowired
     private CaseEventTaskHandler caseEventTaskHandler;
 
     @BeforeEach
     void init() {
         when(mockExternalTask.getTopicName()).thenReturn("test");
         when(mockExternalTask.getWorkerId()).thenReturn("worker");
         when(mockExternalTask.getActivityId()).thenReturn("activityId");
+
         when(mockExternalTask.getAllVariables())
             .thenReturn(Map.of(
-                "CCD_ID",
+                "caseId",
                 CASE_ID.toString(),
-                "CASE_EVENT",
+                "caseEvent",
                 NOTIFY_RESPONDENT_SOLICITOR1_FOR_CLAIM_ISSUE.name()
```

Comment:
as Map could we have each entry on one line? i.e "caseId", CASE_ID

Also CASE_ID should just be string